locales/tr/tr.json

```diff
@@ -1,618 +1,618 @@
-    "January": "January",
-    "Japan": "Japan",
+    "Iran, Islamic Republic Of": "İranlı",
+    "Iraq": "Irak",
+    "Ireland": "İrlanda",
+    "Isle Of Man": "Man Adası",
+    "Israel": "İsrail",
+    "Italy": "İtalya",
+    "Jamaica": "Jamaika",
+    "January": "Ocak",
+    "Japan": "Japonya",
     "Jersey": "Jersey",
-    "Jordan": "Jordan",
-    "July": "July",
-    "June": "June",
-    "Kazakhstan": "Kazakhstan",
+    "Jordan": "Ürdün",
+    "July": "Temmuz",
+    "June": "Haziran",
+    "Kazakhstan": "Kazakistan",
     "Kenya": "Kenya",
-    "Key": "Key",
-    "Kiribati": "Kiribati",
-    "Korea": "South Korea",
-    "Korea, Democratic People's Republic of": "North Korea",
-    "Kosovo": "Kosovo",
-    "Kuwait": "Kuwait",
-    "Kyrgyzstan": "Kyrgyzstan",
-    "Lao People's Democratic Republic": "Laos",
+    "Key": "Anahtar",
+    "Kiribati": "Kiribati. kgm",
+    "Korea": "Güney Kore",
+    "Korea, Democratic People's Republic of": "Kuzey Kore",
+    "Kosovo": "Kosova",
+    "Kuwait": "Kuveyt",
+    "Kyrgyzstan": "Kırgızistan",
+    "Lao People's Democratic Republic": "Laosname",
     "Last active": "Son aktiflik",
     "Last used": "Son kullanım",
-    "Latvia": "Latvia",
+    "Latvia": "Letonya",
     "Leave": "Ayrıl",
     "Leave Team": "Takımdan Ayrıl",
-    "Lebanon": "Lebanon",
+    "Lebanon": "Lübnan",
     "Lens": "Lens",
     "Lesotho": "Lesotho",
-    "Liberia": "Liberia",
+    "Liberia": "Liberya",
     "Libyan Arab Jamahiriya": "Libya",
-    "Liechtenstein": "Liechtenstein",
-    "Lithuania": "Lithuania",
-    "Load :perPage More": "Load :perPage More",
-    "Log in": "Log in",
-    "Log out": "Log out",
-    "Log Out": "Log Out",
-    "Log Out Other Browser Sessions": "Log Out Other Browser Sessions",
+    "Liechtenstein": "Lihtenştayn",
+    "Lithuania": "Litvanya",
+    "Load :perPage More": ":perpage daha fazla yükle",
+    "Log in": "Giriş yap",
+    "Log out": "Çıkış yap",
+    "Log Out": "çıkış yap",
+    "Log Out Other Browser Sessions": "Diğer Tarayıcı Oturumlarını Kapatma",
     "Login": "Giriş Yap",
     "Logout": "Çıkış Yap",
     "Logout Other Browser Sessions": "Diğer Tarayıcılardan Oturumu Kapatın",
-    "Luxembourg": "Luxembourg",
+    "Luxembourg": "Lüksemburg",
     "Macao": "Macao",
-    "Macedonia": "North Macedonia",
-    "Madagascar": "Madagascar",
-    "Malawi": "Malawi",
-    "Malaysia": "Malaysia",
-    "Maldives": "Maldives",
-    "Mali": "Mali",
+    "Macedonia": "Kuzey Makedonya",
+    "Madagascar": "Madagaskar",
+    "Malawi": "Malavi",
+    "Malaysia": "Malezya",
+    "Maldives": "Maldivler",
+    "Mali": "Küçük",
     "Malta": "Malta",
     "Manage Account": "Hesabı Yönet",
-    "Manage and log out your active sessions on other browsers and devices.": "Manage and log out your active sessions on other browsers and devices.",
+    "Manage and log out your active sessions on other browsers and devices.": "Yönetmek ve diğer tarayıcılar ve cihazlar üzerinde etkin oturumları oturum açın.",
```

Comment:
`"Manage and log out your active sessions on other browsers and devices.": "Yönetin ve diğer tarayıcılarda ve cihazlardaki aktif oturumlarınızı kapatın.",`